virtualenv_embedded/distutils-init.py

```diff
@@ -15,7 +14,29 @@
     warnings.warn("The virtualenv distutils package at %s appears to be in the same location as the system distutils?")
 else:
     __path__.insert(0, distutils_path)  # noqa: F821
-    real_distutils = imp.load_module("_virtualenv_distutils", None, distutils_path, ("", "", imp.PKG_DIRECTORY))
+    if sys.version_info < (3, 3):
+        import imp
+
+        real_distutils = imp.load_module("_virtualenv_distutils", None, distutils_path, ("", "", imp.PKG_DIRECTORY))
+    else:
+        import importlib.machinery
+
+        distutils_path = os.path.join(distutils_path, "__init__.py")
+        loader = importlib.machinery.SourceFileLoader("_virtualenv_distutils", distutils_path)
+        if sys.version_info < (3, 4):
+            real_distutils = loader.load_module()
```

Comment:
We don't support anything less than 3.4, so this can be removed.

Reply:
oh I wasn't sure if `-p python3.3` was supported or not, I guess not since it has to exec